ctl/monitoring/monitoring.go

```diff
@@ -46,19 +47,24 @@ func NewCmd() *cobra.Command {
 		Example: `# Enable/Disable Kmesh's accesslog:
 kmeshctl monitoring <kmesh-daemon-pod> --accesslog enable/disable
 
-# Enable/Disable Kmesh's metrics and accesslog:
+# Enable/Disable Kmesh's metrics(to-service and to-pod) and accesslog:
 kmeshctl monitoring <kmesh-daemon-pod> --all enable/disable
 
+# Enable/Disable Kmesh's metrics of connections between pods:
```

Comment:
```suggestion
# Enable/Disable workload granularity metrics:
```

Reply:
what does it mean if <kmesh-daemon-pod> is omitted

Reply:
Will return en error: `Error: unknown flag: --workloadMetric`